articles/quickstart/native/flutter/01-login.md

```diff
@@ -0,0 +1,218 @@
+        // <---
+    }
+}
+```
+
+You do not need to declare a specific `intent-filter` for your activity because you defined the manifest placeholders with your Auth0 **Domain** and **Scheme** values. The library handles the redirection for you.
+
+The sample uses values from `strings.xml`:
+
+- `com_auth0_domain`: The domain of your Auth0 tenant. Generally, you find this in the Auth0 Dashboard under your Application's **Settings** in the Domain field. If you are using a custom domain, you should set this to the value of your custom domain instead.
+- `com_auth0_scheme`: The scheme to use. Can be a custom scheme, or `https` if you want to use [Android App Links](https://auth0.com/docs/applications/enable-android-app-links). You can read more about setting this value in the [Auth0.Android SDK README](https://github.com/auth0/Auth0.Android#a-note-about-app-deep-linking).
+
+Run **Sync Project with Gradle Files** inside Android Studio to apply your changes.
+
+## Configure iOS
+
+If you are not developing for the iOS platform, skip this step.
+
+You need to register your bundle identifier as a custom URL scheme so the callback and logout URLs can reach your app.
+
+In Xcode, go to the **Info** tab of your app target settings. In the **URL Types** section, select the **＋** button to add a new entry. Then enter `auth0` into the **Identifier** field and `$(PRODUCT_BUNDLE_IDENTIFIER)` into the **URL Schemes** field.
+
+<p><img src="/media/articles/native-platforms/ios-swift/url-scheme.png" alt="Custom URL Scheme"></p>
+
+:::note
+The minimum target platform supported by the SDK is **iOS 12**.
+:::
+
+## Add login to your app
+
+[Universal Login](https://auth0.com/docs/authenticate/login/auth0-universal-login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security, and the fullest array of features.
+
+Integrate Auth0 Universal Login in your Flutter app by using the `Auth0` class. Redirect your users to the Auth0 Universal Login page using `webAuthentication().login()`. This is a `Future` and must be awaited for you to retrieve the user's tokens.
+
+See this example of a `ElevatedButton` widget that logs the user in when clicked:
+
+```dart
+ElevatedButton(
+  onPressed: () async {
+    final credentials =
+        await auth0.webAuthentication().login();
+
+    setState(() {
+      _credentials = credentials;
+    });
+  },
+  child: const Text("Log in"))
+```
+
+**Android**: if you are using a custom scheme, pass this scheme to the login method so that the SDK can route to the login page and back again correctly:
+
+```dart
+await auth0.webAuthentication().login(scheme: 'YOUR CUSTOM SCHEME');
+```
+
+When a user logs in, they are redirected back to your application. Then, you are able to access the ID and access tokens for this user.
+
+::::checkpoint
+:::checkpoint-default
```

Comment:
Can these be used in non-interactive QS?

Reply:
Yes they appear to work just fine 👍🏻  I think they were problematic back when the interactive quickstart required some new functionality from the docs platform, but they have been merged now.